### ISSUE TITLE
protected buckets or headers?

### ISSUE THREAD

**Issue by laurencelundblade** [cose-wg/cose-rfc8152bis#20]
The protected buckets are sometimes referred to as headers. 

> The IV can be placed in the unprotected header
> The Parital IV can be placed in the unprotected header
> The parameter is used to indicate which protected header labels
> This attribute is then encoded as an unprotected header.
> Applications SHOULD verify that the same label does not occur in both the protected and unprotected headers

> Headers = (
>     protected : empty_or_serialized_map,
>     unprotected : header_map
> )

**Comment by jimsch** [comment]
The first is already covered in #19.  

I am not sure that there is any benefit in changing the second.

**Comment by laurencelundblade** [comment]
Also, in section 4.4 and 6.3 the protected headers/buckets are referred to as "protected attributes". When I first read this section, I had to do a bit of deduction to convince myself the protected attributes where the protected headers. It seems this important part of COSE should be explicitly clear.

Also, the term "signer structure" could be more explicit by referring to the "signatures" part of the COSE_Sign or some such text.

I searched the document for "protected attribute", "body structure" and "signer structure" to be sure what these were, but the searches didn't help much.

Going back to #17, I think "attribute" is sometimes used to refer to a header parameter and occurrences of it should be checked.
